Assets/BossRoom/Scripts/Shared/Net/UnityRelayUtilities.cs

```diff
@@ -36,6 +35,7 @@ public static async
             try
             {
                 joinCode = await Relay.Instance.GetJoinCodeAsync(allocation.AllocationId);
+                RelayJoinCode.Code = joinCode;
```

Comment:
I'm thinking, this utility could be a good candidate for the utility asmdef. With this in mind, would there be a way to do this without adding gameplay specific code here?

Reply:
RelayJoinCode is a static class containing only the static field Code defined in this same file. It used to be set from GameNetPortal using this same value, but I think it makes more sense to set it here directly and remove the unnecessary clutter. I think we could even remove RelayJoinCode and simply add the join code as a static field of RelayUtility.

Reply:
If we want to keep the static class holding the join code separate though, we could also revert this change, and add a line in ClientGameNetPortal to set RelayJoinCode.Code after the connection succeeded

Reply:
Which solution do you think would make more sense @SamuelBellomo ?

Reply:
as discussed, static field in RelayUtility